bindings/python/manifold3d.cpp

```diff
@@ -568,28 +530,24 @@ NB_MODULE(manifold3d, m) {
            "as empty. Likewise, empty meshes may still show NoError, for "
            "instance if they are small enough relative to their precision to "
            "be collapsed to nothing.")
-      .def_prop_ro(
+      .def(
           "bounding_box",
-          [](Manifold &self) {
-            auto b = self.BoundingBox();
-            nb::tuple box = nb::make_tuple(b.min[0], b.min[1], b.min[2],
-                                           b.max[0], b.max[1], b.max[2]);
-            return box;
+          [](Manifold const &self) {
+            Box b = self.BoundingBox();
+            return nb::make_tuple(b.min[0], b.min[1], b.min[2], b.max[0],
+                                  b.max[1], b.max[2]);
           },
           "Gets the manifold bounding box as a tuple "
           "(xmin, ymin, zmin, xmax, ymax, zmax).")
       .def_static(
           "smooth",
           [](const MeshGL &mesh,
-             const std::vector<std::tuple<int, float>> &sharpenedEdges = {}) {
-            std::vector<Smoothness> vec(sharpenedEdges.size());
-            for (int i = 0; i < sharpenedEdges.size(); i++) {
-              vec[i] = {std::get<0>(sharpenedEdges[i]),
-                        std::get<1>(sharpenedEdges[i])};
-            }
+             std::vector<std::pair<int, float>> sharpenedEdges) {
```

Comment:
I'm thinking if we want to make this numpy-friendly, maybe by splitting it into an integer index array and smoothness array.

Reply:
Makes sense to me. Done